orchestrator_mapreduce.py

```
@@ -0,0 +1,380 @@
+        Execute the full Plan → Map → Reduce cycle.
+        This is the single entry point.
+        """
+        job = await self.plan(intent, task_list)
+        job = await self.map_execute(job)
+        job = await self.reduce_verify(job)
+        return job
+
+    # ─── Internal Helpers ──────────────────────────────────────
+
+    async def _run_protocol_isolated(
+        self, protocol_name: str, inputs: Dict[str, Any]
```

Ruff (0.15.18)

[warning] Unused method argument: `inputs`

(ARG002)